pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -270,11 +270,15 @@ metrics:
     enabled: true # ENV: KUMA_METRICS_DATAPLANE_ENABLED
     # How many latest subscriptions will be stored in DataplaneInsight object, if equals 0 then unlimited
     subscriptionLimit: 2 # ENV: KUMA_METRICS_DATAPLANE_SUBSCRIPTION_LIMIT
+    # How long data plane proxy can stay Online without active xDS connection
+    idleTimeout: 5m # ENV: KUMA_METRICS_DATAPLANE_IDLE_TIMEOUT
   zone:
     # Enables collecting metrics from Zone
     enabled: true # ENV: KUMA_METRICS_ZONE_ENABLED
```

Comment:
I'm thinking... now we rely on Insights to provide a stable product. Should we remove an option to disable insights?